language-design.md

```diff
@@ -43,6 +82,11 @@ In what way is an API a language?
 
 **Response**
 
+- Good API's get reused
+- Names matter
+- People must learn to read and write it
+- Should be easy to use
+
```

Comment:
I know the question didn't explicitly ask, but it would have also been interesting to hear your ideas on what ways an API is _not_ a language.